openforcefield/topology/molecule.py

```diff
@@ -111,29 +108,29 @@ def molecule_particle_index(self):
         """
         return self._molecule.particles.index(self)
 
-    @property
-    def topology(self):
-        """
-        The ``Topology`` this atom is part of.
-
-        .. todo::
-
-           * Should we have a single unique ``Topology`` for each topology type in the system,
-           or if we have multiple copies of the same topology, should we have multiple ``Topology``s?
-        """
-        return self._topology
-
-    @topology.setter
-    def topology(self, topology):
-        """
-        Set the particle's topology pointer. Note that this will only work if the particle currently 
-        doesn't have a topology
-        """
-        #TODO: I forgot if python will hold a pointer to the molecule, or some sort of bulky copy here
-        #TODO: Should _molecule be a property?
-        #TODO: Add informative exception here
-        assert self._topology == None
-        self._topology = topology
+    # @property
```

Comment:
Why is this commented out? Shouldn't we either delete it or add a `NotImplementedException`?

Reply:
I didn't delete this because I wanted to figure out the origin of this code. In the current data structure, `openforcefield.topology.Atom` objects do not belong to a `Topology`. Only `openforcefield.topology.TopologyAtom` objects do.

This code might be left over from a previous design era, or it might be Andrea's start at an attempt to unify `Particle` and `TopologyParticle`. We should meet to clear up any ambiguity about our data structure.

Reply:
Opened #235 so developers can make sure we're on the same page about this.

Reply:
Old code was mine---definitely OK to delete!

Reply:
Gotcha. Removed it in 3f184a1  